src/tests/mod.rs

```diff
@@ -285,6 +285,28 @@ mod checkout {
     pub(crate) fn checkout_new_branch() {
         snapshot!(TestContext::setup_clone(), "bcf<esc>bcx<enter>");
     }
+
+    #[test]
+    pub(crate) fn delete_branch_menu() {
+        let ctx = TestContext::setup_clone();
+        run(ctx.dir.path(), &["git", "branch", "branch-to-delete"]);
+        snapshot!(ctx, "YjbDy");
+    }
```

Comment:
I think a lot of other similar tests are structured like:

```suggestion

    #[test]
    pub(crate) fn delete_branch_prompt() {
        let ctx = TestContext::setup_clone();
        run(ctx.dir.path(), &["git", "branch", "branch-to-delete"]);
        snapshot!(ctx, "YjbD");
    }

    #[test]
    pub(crate) fn delete_branch_menu() {
        let ctx = TestContext::setup_clone();
        run(ctx.dir.path(), &["git", "branch", "branch-to-delete"]);
        snapshot!(ctx, "YjbDy");
    }
```